apps/aecore/src/aec_sync.erl

```diff
@@ -414,52 +365,43 @@ process_job([{_T, Job}]) ->
             do_server_get_missing(Uri);
         {fetch_mempool, Uri} ->
             do_fetch_mempool(Uri);
-        {ping, Uri} ->
-            await_aehttp_and_ping_peer(Uri);
+        {ping, PeerId} ->
+            %% ping_peer(PeerId);
+            await_aehttp_and_ping_peer(PeerId);
         _Other ->
             lager:debug("unknown job", [])
     end.
 
 enqueue(Op, Msg) ->
-    [ jobs:enqueue(sync_jobs, {Op, Msg, Uri}) || Uri <- aec_peers:get_random(all) ].
+    Peers = aec_peers:get_random(all),
+    lager:debug("ZZPC: enqueue ~p to ~p", [Op, Peers]),
+    [ jobs:enqueue(sync_jobs, {Op, Msg, aec_peers:peer_id(PeerInfo)})
+      || PeerInfo <- Peers ].
 
-await_aehttp_and_ping_peer(Uri) ->
+await_aehttp_and_ping_peer(PeerId) ->
     %% Don't ping until our own HTTP endpoint is up. This is not strictly
     %% needed for the ping itself, but given that a ping can quickly
     %% lead to a greater discovery, we should be prepared to handle pings
     %% ourselves at this point.
     %% The URI comes from aec_peers and is a valid http url.
     case await_aehttp() of
```

Comment:
Should we really still wait for aehttp?

Reply:
No, I don't think we need to do that anymore... Probably just a leftover from before!

Reply:
DONE